python/lsst/afw/cameraGeom/camera.py

```diff
@@ -106,8 +129,38 @@ def findDetectors(self, cameraPoint):
                 detectorList.append(detector)
         return detectorList
 
+    def findDetectorsArray(self, xCoord, yCoord, coordSys):
+        """!Find the detectors that cover a list of points specified by x and y coordinates in any system
+
+        @param[in] xCoord is an array of x coordinates
+        @param[in] yCoord is an array of y coordinates
+        @param[in] coordSys is the coordinate system in which xCoord and yCoord are defined
+        @return a list of lists; each list contains the names of all detectors which contain the
+        corresponding point
+        """
+
```

Comment:
For a similar reason, I'm going to push back on Russell's comment, since, in CatSim, we already have numpy arrays of x and y.  Converting them into Point2D in CatSim (which is all python), is what was slowing us down before.